contracts/SportMarkets/Rundown/GamesOddsObtainer.sol

```diff
@@ -258,14 +258,27 @@ contract GamesOddsObtainer is Initializable, ProxyOwned, ProxyPausable {
     /// @notice function which retrievers all markert addresses for given parent market
     /// @param _parent parent market
     /// @return address[] child addresses
-    function getAllChildMarketsFromParent(address _parent) public view returns (address[] memory) {
+    function getAllChildMarketsFromParent(address _parent) external view returns (address[] memory) {
         address[] memory allMarkets = new address[](numberOfChildMarkets[_parent]);
         for (uint i = 0; i < numberOfChildMarkets[_parent]; i++) {
             allMarkets[i] = mainMarketChildMarketIndex[_parent][i];
         }
         return allMarkets;
     }
 
+    /// @notice function which retrievers all markert addresses for given parent market
+    /// @param _parent parent market
+    /// @return totalsMarket totals child address
+    /// @return spreadsMarket spread child address
+    function getActiveChildMarketsFromParent(address _parent)
+        external
+        view
+        returns (address totalsMarket, address spreadsMarket)
+    {
+        totalsMarket = currentActiveTotalChildMarket[_parent];
```

Comment:
why not using public variables currentActive...?

Reply:
to make single external call instead of two calls per market